chore(grid odata):add note for select parameter (#470)
telerik/blazor-docs: knowledge-base/grid-odata.md

## Solution

An example is available in the following project: [https://github.com/telerik/blazor-ui/tree/master/grid/odata](https://github.com/telerik/blazor-ui/tree/master/grid/odata)

> In this example application, the ODATA query contains all the available data. This configuration is intentional in order to cover different scenarios. For example, columns can be conditional, some may be calculated, and we don't know what else that data might be used for in the specific case, some logic may even rely on fields that are not shown in the grid.

Reviewer: Replace listing scenarios in long sentences with bullet lists to improve readability:

```suggestion
In this example application, the ODATA query contains all the available data. This configuration is intentional in order to cover different scenarios. For example:
* some columns may be calculated;
* some columns may be conditionally displayed;
* some column templates may rely on fields that are not shown in the grid;
```
